app/src/interfaces/map/map.vue

```diff
@@ -548,3 +548,9 @@ function handleKeyDown(event: any) {
 	opacity: 0;
 }
 </style>
+
+<style lang="scss">
+.mapbox-gl-draw_ctrl-draw-btn {
+	color: unset;
+}
+</style>
```

Comment:
I think this should be integrated into the already existing https://github.com/directus/directus/blob/main/app/src/styles/lib/_mapbox.scss